styles/styles.css

```diff
@@ -0,0 +1,248 @@
+}
+
+.home-container sub {
+  height: 2rem;
+  font-family: 'PT Sans', sans-serif;
+  font-size: 2rem;
+}
+
+#call-action {
+  font-family: 'PT Sans', sans-serif;
+  font-size: 1rem;
+  height: 1.5rem;
+  color: black;
+  border: 1px solid grey;
+  border-radius: 8px;
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 1.2rem;
+  background-color: white;
+}
+
+.home-container {
+  background-image: "trianglify.png"; 
+  background: linear-gradient(white, aqua);
+  height: 48rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-bottom: 3em;
+  border-bottom: solid thin gray;
+}
+
+.quote-grid {
+  /* margin-left: 8rem;
+  margin-right: 8rem; */
+  display: grid;
+  height: 15rem;
+  grid-template-rows: 8rem;
+  grid-template-columns: 20% 60% 20%;
+  border-bottom: solid thin gray;
+}
+
+.quote {
+  grid-row: 1;
+  grid-column: 2;
+  font-style: italic;
+  font-size: 1.4rem;
+  text-align: left;
+  font-family: 'Quicksand', sans-serif;
+  padding-top: 2rem;
+}
+
+.author {
+  grid-row: 2;
+  grid-column: 2;
+  font-size: 1.2em;
+  text-align: right;
+  font-family: 'Quicksand', sans-serif;
+}
+
+.why-grid {
+  background-color: aqua;
+  justify-content: center;
+  width: 100%;
+  font-family: 'Quicksand', sans-serif;
+  height: 20rem;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  /* background-color: blueviolet; */
+  padding-bottom: 4rem;
+  border-bottom: solid thin gray;
+  /* grid-template-columns: 1 1 1; */
+}
+
+#why-element {
+  grid-row: 1;
+  text-align: center;
+  font-weight: bold;
+  font-size: 2rem;
+}
+
+.images-container {
+  grid-row: 2;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 100%;
+  /* grid-gap: 4rem; */
+}
+
+.image-item {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  width: 300px;
+  /* grid-template-rows: 1fr 1fr; */
+}
+
+.image-item img {
+  grid-column: 1;
+  grid-row: 1 / span 2
+}
+
+.image-item strong {
+  grid-column: 2;
+  grid-row: 1;
+  font-size: 1.2rem;
+}
+
+.image-item p {
+  padding-top: 0.5rem;
+  grid-column: 2;
+  grid-row-start: 1;
+}
+
+.team-members {
+  font-family: 'Quicksand', sans-serif;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: repeat(7, 1fr)
+}
+
+#meet-team {
+  font-family: 'Quicksand', sans-serif;
+  grid-row: 1;
+  text-align: center;
+  font-weight: bold;
+}
+
+#team-1 {
+  grid-column-start: 2;
+  grid-row-start: 1;
+}
+
+#team-2 {
+  grid-column: 4;
+  grid-row: 1;
+}
+
+#team-3 {
+  grid-column: 6;
+  grid-row: 1;
+}
+
+#team-4 {
+  grid-column: 3;
+  grid-row: 2;
+}
+
+#team-5 {
+  grid-column: 5;
+  grid-row: 2;
+}
+
+.team-members span {
+  display: block;
+  text-align: center;
+}
+
+.team-members li {
+  list-style: none;
+}
+
+.team-members img {
+  border-radius: 100%;
+  padding-bottom: 1rem;
+}
+
+.sponsors-grid {
```

Comment:
Were you intending for your sponsors to be touching the line that separates them from the footer? If not, I think you can add a padding-bottom property with a value of 5rem or higher, and it should add some white space.